test/__init__.py

```diff
@@ -5,14 +5,16 @@
 import socket
 import sys
 import warnings
+from types import ModuleType
+from typing import TYPE_CHECKING, Any, Callable, Dict, List, Optional
 
 import pytest
 
 try:
     try:
-        import brotlicffi as brotli
+        import brotlicffi as brotli  # type: ignore
```

Comment:
Suggestion (take it or leave it): When using `type: ignore`, I prefer to also specify the error code, e.g. `type: ignore[import]` (the error code is mentioned in the mypy error). This way, if the reason for the error changes, the ignore no longer works, which is good. It also helps the reader understand what is being ignored.

Reply:
Good suggestion. I've updated all `type: ignore` in this path.
I will create a patch to update all other `type: ignore`.
Or maybe it can be a good contributor-friendly issue.
what do you think @sethmlarson

Reply:
It'd be great if we could have one person go through and commit all of these at once so probably not great for a new contributor. Could you handle these?

Reply:
Sure. I will create a PR for this